### PR TITLE
Tests for Add and Remove tasks

### PR DESCRIPTION
**Pull request by yaseerokino**
## Features Added ♻️ 
- Added test function to test adding a task 
- Added test function to test removing a task

Note: My partner for this week has not been available and when i contacted him, he told me he plans on reapeating the week, so I had to do the project myself

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi Team,

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Tests are grouped using the describe() method.
:+1: All functions responsible for adding and deleting list items are tested.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] README file should have steps about how to test the project. (Hint: `npm run test`)
- [ ] I would recommend removing unnecessary files like `.DS_Store` to keep the project structure clean and professional.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @yaseerokino , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/task.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { addTask, removeTask } from '../module/task';
+
+jest.mock('../module/storage.js');
+
+let tasks = [
+  { description: 'do some coding', completed: false, index: 1 },
+  { description: 'hire a help', completed: true, index: 2 },
+];
+
+describe('Add Tasks', () => {
```
Comment (Rustamxon7) [inline src/test/task.test.js:10]: Try to add a more descriptive description for your project. It helps testers to understand about your test is for.

**Review by Rustamxon7 — COMMENTED** [review]
Your project is looking great👍. 

But there is an optional change...

Check out this comment.

**Pull request merged**